### ISSUE TITLE
ImportError: No module named 'requests.packages.chardet.sys' when frozen by PyInstaller

### ISSUE THREAD

**Issue by htgoebel** [kennethreitz/requests#2583]
When running a simple test-program frozen by [PyInstaller](http://www.pyinstaller.org) this program fails with an ```ImportError: No module named 'requests.packages.chardet.sys'```. The same program runs fine when run unfrozen (which means: directly from the command line).

This seams to be a problem with the import magic done in requests.packages. For some reasons requests.packages.chardet tries a relative import.

How to reproduce:
```
$ git clone --depth=1 https://github.com/pyinstaller/pyinstaller.git
$ cd pyinstaller
# the script runs unfrozen (well, it has an SSL-error, but no import error):
$ python tests/libraries/test_requests.py
# now freeze the application
$ cd tests/libraries/
$ ../../pyinstaller.py test_requests.py
# the frozen app fails:
$ dist/test_requests/test_requests
[...]
  File "/tmp/pyinstaller/PyInstaller/loader/pyi_importers.py", line 276, in load_module
    exec(bytecode, module.__dict__)
  File "/tmp/pyinstaller/tests/libraries/build/test_requests/out00-PYZ.pyz/requests.packages.chardet", line 19, in <module>
  File "/tmp/pyinstaller/tests/libraries/build/test_requests/out00-PYZ.pyz/requests.packages", line 95, in load_module
ImportError: No module named 'requests.packages.chardet.sys'
```

Tested with requests __version__ = '2.6.0', __build__ = 0x020503 and Python 2.7.9

**Comment by Lukasa** [comment]
This is a known issue in 2.6.0 with the update machinery we introduced in 2.5.2. We fixed it in 2.6.1, but unfortunately that release also broke some chunked encoding stuff. The reality is that there are no good releases right now for pyinstaller (2.1.0 through 2.5.3 are affected by CVE 2015-2296, so don't use them either), but 2.7.0 should be along shortly which will resolve this.

**Comment by htgoebel** [comment]
@Lukasa Thanks for the quick reply. I can confirm the error is gone in 2.6.1.
